website/docs/reference/options.mdx

```diff
@@ -258,11 +258,11 @@ Not all features of an `http.Request` are currently supported, please see the fo
 | DELETE                  | ✅  | ✅  | ✅ [^1] |
 | Request Headers         | ✅  | ✅  | ✅ [^1] |
 | Request Body            | ✅  | ✅  | ❌      |
-| Request Body Streaming  | ❌  | ❌  | ❌      |
+| Request Body Streaming  | ✅  | ✅  | ❌      |
```

Comment:
Was Windows updated as well?

Reply:
We already supported that on Windows for quite some time. Just the docs wasn't updated 😄

Reply:
Response Streaming for Windows will come soon 😄